### ISSUE TITLE
Kiwix blog atom/rss feed

### ISSUE THREAD

**Issue by kelson42** [kiwix/web#118]
There is no feed for the moment at https://www.kiwix.org/blog/.
I have no way to be informed that something new has been posted there for the moment.
An atom or rss feed would solve the problem.
This should be doable really easily I guess via a Wordpress option/plugin.

**Comment by kelson42** [comment]
BTW, I thought we had one! Maybe it has disappeared?

**Comment by martin8032** [comment]
As far as I know you simply add /feed/ to a WordPress URL to get the feed.

Is https://www.kiwix.org/feed/ what you are looking for?

**Comment by kelson42** [comment]
@martin8032 None of what I have tried has worked. The response to the URL you have given to me has an empty body and is not a news feed.

**Comment by martin8032** [comment]
You are right, but that's weird. This is the way to access the RSS feed on any WordPress website (try https://www.metoki.ch/feed/ for example). But you should not get a body tag at all, since this should be XML, not HTML. I don't have an explanation/solution at the moment.

**Comment by mutante** [comment]
Hi, i am maintaining the Wikimedia Planet RSS Feed aggregator (https://meta.wikimedia.org/wiki/Planet_Wikimedia  , https://en.planet.wikimedia.org/).

I had to remove the Kiwix blog from it because at some point there was no more working RSS or Atom feed (it was in our configs so it must have existed in the past).

I can also confirm there are many other Wordpress-based sites that _do_ provide a feed.

Would be more than happy to be able to re-add Kiwix blog if this was fixed. 

Maybe somebody would want to contact Metoko, because apparently they are making the site https://www.metoki.ch/kontakt/

**Comment by martin8032** [comment]
Hi @mutante, thank you for contacting us. I am the owner of Metoki, and as you can read above we are already aware of this problem. We have just made some changes to the site, so maybe you could test again and check if our feed is available again? Thank you.

**Comment by kelson42** [comment]
@martin8032 There is still no feed. What is needed here is:
* A working feed announcing the blog posts in atom or RSS format (you need to tell us the URL(s))
* This URL(s) put in the blog HTML pages with `<link>` DOM nodes like explained here https://riptutorial.com/html/example/4367/web-feed
* optional but nice: a feed icon somwhere visible by user on the blog HTML pages

**Comment by kelson42** [comment]
has been fixed in new web site
